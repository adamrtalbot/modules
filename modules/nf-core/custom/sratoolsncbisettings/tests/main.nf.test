nextflow_process {

    name "Test Process CUSTOM_SRATOOLSNCBISETTINGS"
    script "modules/nf-core/custom/sratoolsncbisettings/main.nf"
    process "CUSTOM_SRATOOLSNCBISETTINGS"

    test("Should run without failures") {

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
