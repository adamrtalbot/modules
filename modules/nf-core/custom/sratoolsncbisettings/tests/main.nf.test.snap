{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "user-settings.mkfg:md5,8733bcd475a88fb919139573602605ba"
                ],
                "1": [
                    "versions.yml:md5,fec13b593c3b42ddd38f2fc77df25b70"
                ],
                "ncbi_settings": [
                    "user-settings.mkfg:md5,8733bcd475a88fb919139573602605ba"
                ],
                "versions": [
                    "versions.yml:md5,fec13b593c3b42ddd38f2fc77df25b70"
                ]
            }
        ],
        "timestamp": "2023-10-11T18:07:25.95687"
    }
}